js/crm.ajax.js

```diff
@@ -498,7 +498,7 @@
       settings = $el.data('popup-settings') || {},
       formData = false;
     settings.dialog = settings.dialog || {};
-    if (e.isDefaultPrevented() || !CRM.config.ajaxPopupsEnabled || !url || $el.is(exclude)) {
+    if (e.isDefaultPrevented() || !CRM.config.ajaxPopupsEnabled || !url || $el.is(exclude + ', .no-open')) {
```

Comment:
This should not be necessary per my other comment.

Reply:
If we want to use no-popup above, we should remove it from the exclude array and instead explicitly exclude it here like I'm doing with no-open. I didn't want to do this since it's used in many places in the code base but in my randomized testing just now it did not seem to be used to close an existing pop up window in the code base (which is what it does in the other place exclude is used).